samples/line-time-point-data.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
     <title>Time Scale Point Data</title>
-    <script src="../node_modules/moment/min/moment.min.js"></script>
+    <script src="../bower_components/moment/min/moment.min.js"></script>
```

Comment:
Hmm sure. I have recently entered the world of npm and bower and like you said if npm install is being run by default, bower is probably unnecessary.